contracts/Claims.sol

```diff
@@ -834,11 +834,6 @@ contract Claims is AccessControlEnumerable {
                 revert Balance("below minimum", min);
             }
 
-            // Ensure anyone can stake up until the defined expiry threshold.
-            if (_claimExpired[cla][CLAIM_EXPIRY_T] < block.timestamp) {
-                revert Expired("staking over", _claimExpired[cla][CLAIM_EXPIRY_T]);
-            }
```

Comment:
This moved down because if we staked on a claim that does not exist we would get the error "staking over". Now we get "claim invalid" because the existence check runs first.